gcp_variant_transforms/libs/bigquery_vcf_schema_converter.py

```diff
@@ -227,14 +227,15 @@ def _add_format_fields(schema, formats, allow_incompatible_schema=False):
           id=field.name,
           num=reserved_definition.num,
           type=reserved_definition.type,
-          desc=field.description or reserved_definition.desc)})
+          desc=(_remove_new_line_character(field.description) or
```

Comment:
nit: it's a bit cleaner to just do `desc=_remove_special_characters(field.description or reserved_definition.desc)`...we know that our reserved defs dont include special characters so it's a no-op.